### PR TITLE
add: `protonvpn-gui`

### PR DESCRIPTION
**Pull request by WRM-42**
- [x] Edit packagelist
- [x] Add initial pacscript
- [ ] ~Contact devs (via email)~
- [x] Add maintainer to pacscript

### CODE REVIEW

File: packages/protonvpn-gui/protonvpn-gui.pacscript [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name="protonvpn-gui"
+gives="protonvpn"
+breaks="${name}-git"
```
Comment (Henryws) [inline packages/protonvpn-gui/protonvpn-gui.pacscript:3]:
```suggestion
breaks="${name}-git ${name}-bin"
```

**Review by Henryws — CHANGES_REQUESTED** [review]

**Review by Henryws — APPROVED** [review]

**Review by Henryws — CHANGES_REQUESTED** [review]
CI fails.

**Comment by WRM-42** [comment]
The error seems to be CI specific, everything works fine on a normal machine

https://packages.debian.org/buster/gir1.2-appindicator3-0.1

**Comment by cat-master21** [comment]
Because: https://www.debian.org/releases/bullseye/armhf/release-notes.en.pdf se section `5.3.1 Noteworthy obsolete packages` 

> The deprecated libappindicator libraries are no longer provided. As a result, the related pack-
>
>ages libappindicator1, libappindicator3-1 and libappindicator-dev are no longer
>available. This is expected to cause dependency errors for third-party software that still depends
>on libappindicator to provide system tray and indicator support.
the same issue with discord package.

**Review by Henryws — APPROVED** [review]

**Pull request merged**